tsconfig.json

```diff
@@ -3,10 +3,14 @@
 	"compilerOptions": {
 		"outDir": "./dist",
 		"lib": [
-			"ES2023"
+			"ES2024",
+			"ESNext.Array",
+			"ESNext.Collection",
+			"ESNext.Iterator",
+			"ESNext.Promise"
```

Comment:
From https://www.npmjs.com/package/@tsconfig/node24

Reply:
We still must support 18 as a minimum so these settings shouldn't change.